tests/test_plugins/test_crypto_plugins/keys/dsa-key.pub

```diff
@@ -0,0 +1 @@
+ssh-dss AAAAB3NzaC1kc3MAAACBANTcOLxTjiR+AyAKJvsLNbJlXrfxBbZryKk5qacc9jEONOink4DdSsqJBwYBAD7RqF7luT3vc+8nyPzkxR9yG+OJ1WAxoBdJBs8ew184K8Ne0Jo6Sj0vkv2FSazp6f+l6tBkXdvyeIuwk4n9Ky+SwVCQCDyqJ5YKB+X4chvkzkzJAAAAFQCcRnwbjfb26nzLVuV6BNpXS+jk/wAAAIB8efzMiNQYnmhxMElS53dOSX07ghgvmKySVTP0S5Mm/xdlZ5Va3TpYr9AYJBWr2dBq+PjyFN5Tv8X9XtILkEOxm/zMsx2VOiu0fNSj5uFXwxYhm+ypLyCX0GACbs39Olb83lyqGwtIYwBiXW1jtlFdQ5AzbHjkfgONxC6Rgm+F6AAAAIAkqv1WdLA24/BwGf0CcGqX+ORKV1ymjU3x43WJH/0mR8cs4wDt1v76vkBj/Vrv8C6X7x4eU1ZG8TvqIq4eRJl2vk4wG5rHZ1ttNwQHDW1InsxRQYGaio4aYVEC2kJHrKA/CoMsPuNaC8ZeaTNFoyG6ne7uB0RyyuS0z49cq/YNAg== dhaugh@dhaugh-ld2
```

Comment:
Remove your user and host from these files and replace with something generic. I'm guessing these keys aren't for anything sensitive, but can you confirm that?

Reply:
Generated just for this purpose - removed the usernames and host